src/test/java/com/axonivy/ivy/supplements/logviewer/parser/TestLogFileParser.java

```diff
@@ -13,11 +14,18 @@
 	/*
 	 * This test tests the current state of parsing, which is not perfect yet.
 	 */
+
+	//TODO: reduce amount of asserts per test in this class
```

Comment:
Will this TODO still be done or just forgot to remove?

Reply:
Imo if this is done, then as part of an other story. It kinda depends on how you want to write your tests. Imo there should be one assert only per test, as the test should only fail for one reason. However after researching I came to the conclusion that this is more of a preference-thing.

Reply:
I'd just remove the TODO, we normally don't bother when it has more than one assert. Also I don't think that the project will have too big tests in the next few years.